step-templates/msmq-create-transactional-queue.json

```diff
@@ -1,70 +1,105 @@
+    "Octopus.Action.Script.Syntax": "PowerShell",
+    "Octopus.Action.Script.ScriptSource": "Inline",
+    "Octopus.Action.RunOnServer": "false",
+    "Octopus.Action.Script.ScriptFileName": null,
+    "Octopus.Action.Package.FeedId": null,
+    "Octopus.Action.Package.PackageId": null
   },
-  "SensitiveProperties": {},
   "Parameters": [
     {
+      "Id": "35154cac-d005-4a7b-85c9-8eab276e726b",
       "Name": "$MSMQQueues",
       "Label": "Queue names",
       "HelpText": "Queue names, separated by semicolons. Example: _Queue1;Queue2;Queue3_",
       "DefaultValue": "",
-      "DisplaySettings": {}
+      "DisplaySettings": {},
+      "Links": {}
     },
     {
+      "Id": "6f3a90b7-df04-4884-9cd1-f04ad7a1f97b",
       "Name": "$MSMQUsers",
       "Label": "Queue users",
       "HelpText": "Users with access to the queue separated by semicolons. Example: _DOMAIN\\User1;DOMAIN\\User2_",
       "DefaultValue": "",
-      "DisplaySettings": {}
+      "DisplaySettings": {},
+      "Links": {}
     },
     {
+      "Id": "3e2e1512-6f75-4e36-80cc-d350a3563a09",
       "Name": "$MSMQPermAllow",
       "Label": "Allowed permissions",
       "HelpText": "Permissions granted to the queue users, separated by semicolons. Example: _DeleteMessage;PeekMessage;ReceiveMessage_",
       "DefaultValue": "DeleteMessage;PeekMessage;ReceiveMessage;WriteMessage",
-      "DisplaySettings": {}
+      "DisplaySettings": {},
+      "Links": {}
     },
     {
       "Name": "$MSMQPermDeny",
       "Label": "Denied permissions",
       "HelpText": "Denied permissions, separated by semicolons: _TakeQueueOwnership_",
       "DefaultValue": "TakeQueueOwnership",
-      "DisplaySettings": {}
+      "DisplaySettings": {},
+      "Links": {}
     },
     {
       "Name": "MSMQAdminUsers",
       "Label": "Admin queue users",
       "HelpText": "Users with access to the queue separated by semicolons. Example: _DOMAIN\\User1;DOMAIN\\User2_",
       "DefaultValue": null,
-      "DisplaySettings": {}
+      "DisplaySettings": {},
+      "Links": {}
     },
     {
       "Name": "MSMQPermAdminAllow",
       "Label": "Allowed admin permissions",
       "HelpText": "Permissions granted to the queue admin users, separated by semicolons. Example: _DeleteMessage;PeekMessage;ReceiveMessage_",
       "DefaultValue": "FullControl",
-      "DisplaySettings": {}
+      "DisplaySettings": {},
+      "Links": {}
     },
     {
       "Name": "MSMQPermAdminDeny",
       "Label": "Denied admin permissions",
       "HelpText": "Denied permissions, separated by semicolons: _TakeQueueOwnership_",
       "DefaultValue": null,
-      "DisplaySettings": {}
+      "DisplaySettings": {},
+      "Links": {}
+    },
+    {
+      "Name": "$resetPermissions",
```

Comment:
Hrm, my apologies re: the `$` in parameter names, I (incorrectly) assumed it would cause a few headaches but it clearly doesn't. And, yes just removing it from the variables you've added is fine 👍